File: chatwoot/python/src/UpdateAgentBotExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
    # api_key={"agentBotApiKey": "AGENT_BOT_API_KEY"},
    # api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    update_agent_bot_request = models.UpdateAgentBotRequest(
        agent_bot=0,
    )

    try:
        api.InboxesApi(api_client).update_agent_bot(
            account_id=0,
            id=0,
            data=update_agent_bot_request,
        )
    except ApiException as e:
        print("Exception when calling InboxesApi#update_agent_bot: %s\n" % e)

File: chatwoot/ruby/src/UpdateAgentBotExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

update_agent_bot_request = ChatwootClient::UpdateAgentBotRequest.new
update_agent_bot_request.agent_bot = 0

begin
    ChatwootClient::InboxesApi.new.update_agent_bot(
        0, # account_id
        0, # id
        update_agent_bot_request, # data
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling InboxesApi#update_agent_bot: #{e}"
end
